src/components/xDragBoxLayer.ts

```diff
@@ -17,7 +17,7 @@ export module Components {
 
     public computeLayout(origin?: Point, availableWidth?: number, availableHeight?: number) {
       super.computeLayout(origin, availableWidth, availableHeight);
-      this.bounds(this.bounds()); // set correct bounds when width/height changes
+      this._setBoundsWithoutChangingMode(this.bounds()); // set correct bounds when width/height changes
```

Comment:
Code like the one that was previously here smells `this.bounds(this.bounds())`. Can we use the appropriate protected / private method for the invalidation of old bounds? 

Current change is just a rewriting of what was previously there (and wrong)